docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+---
+sidebar_position: 2
+title: Tutorial
+tags: [quickstart]
+---
+
+:::info
+
+This tutorial doesn’t assume any existing knowledge of the blindnet devkit or even any privacy-related technologies.
+
+:::
+
+The purpose of the following tutorial is to serve as an entry point for the blindnet devkit, by simply walking you through its different components to give you a better understanding of the overall structure and logic.
+
```

Comment:
If we indeed assume no prior knowlegde about the devkit, then before the next paragraph (where we tell them that we'll aswer questions on the purpose of Privacy Computation Engine) it seams appropriate to add here a brief overview of what the components are, or at least list them.

Reply:
Maybe even an image showing all the components would be appropriate here.

Reply:
I am indeed writing a better overview explainer as we speak (and it does include a graph).

However, I didn't decide if it should be part of the main introduction or a separate document yet (it will depend on how long it has to be).

I consider the introductory tutorial should just give a "quick pick" overview, so I especially want to avoid going too deep into conceptual and technical details. It shouldn't take more than 10 minutes to be completed, or max 20 minutes if absolutely necessary

As soon as the devkit is ready enough, the introductory tutorial will only be a "learn by practice" guide (to create a first complete "privacy stack" step by step, using all components), while more advanced tutorials will allow going deeper into each component separately.